vespajlib/src/main/java/com/yahoo/tensor/TensorType.java

```diff
@@ -135,6 +142,9 @@ public static TensorType fromSpec(String specString) {
     /** The type representing the mapped subset of dimensions of this. */
     public TensorType mappedSubtype() { return mappedSubtype; }
 
+    /** The type representing the indexed subset of dimensions of this. */
+    public TensorType indexedSubtype() { return indexedSubtype; }
```

Comment:
@bratseth Consider adding test for this (and mappedSubtype()) in TensorTypeTestCase.java.

Reply:
will do